pkg/client/didexchange/client.go

```diff
@@ -109,7 +109,7 @@ func (c *Client) CreateInvitation(label string) (*Invitation, error) {
 		Label:           label,
 		RecipientKeys:   []string{verKey},
 		ServiceEndpoint: c.inboundTransportEndpoint,
-		Type:            didexchange.ConnectionInvite,
+		Type:            didexchange.DIDExchangeInvite,
```

Comment:
`didexchange.DIDExchange`?

Reply:
valid point. Will updated to Invite.

Reply:
[Updated](https://github.com/hyperledger/aries-framework-go/pull/576/files#diff-427522b353a28b1ec350c29c79d387b0R112). Added prefix "MsgType" as name collide with message structs.